sdks/go/test/integration/io/mongodbio/mongodbio_test.go

```diff
@@ -0,0 +1,232 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbio
```

Comment:
I have excluded the mongodbio integration tests to be run from **dataflowValidatesRunner** since they use local test containers. For the other ValidatesRunner tasks, I run into various sorts of issues when trying to run them locally:
- **flinkValidatesRunner** - unable to run any job, getting a `java.lang.OutOfMemoryError: Could not allocate enough memory segments for NetworkBufferPool`
- **samzaValidatesRunner** - unable to run any job, getting a `java.util.concurrent.CompletionException: java.lang.IncompatibleClassChangeError: Method 'void org.apache.samza.storage.kv.KeyValueStore.deleteAll(java.util.List)' must be InterfaceMethodref constant`
- **sparkValidatesRunner** - able to run jobs. The Read and Write transforms for mongodbio work if the document id is of type `string`, but not `primitive.ObjectID`: `panic: reflect.Value.Bytes of unaddressable byte array`
- **ulrValidatesRunner** - unable to run any job, there seems to be some architecture incompatibility issues for importing some Python packages (I use a Mac M1) so the job service does not appear to start up

I'm not sure what to conclude from this. I have a hard time debugging what is going on with the runners and would need some help with understanding whether I should change anything in the implementation, or if there is lack of support in some runners so that they should be filtered out. And regarding the SparkRunner, I have registered a coder for `primitive.ObjectID` and can run successfully with DataflowRunner. Does the SparkRunner not support custom coders or have I not implemented or registered it correctly?

Reply:
Cool, looking forward to trying it out!